pykit/ClCommand.py

```diff
@@ -11,7 +11,7 @@ def __init__(self, command, screen_output=False):
 
     def get_payload(self):
         if self.screen_output:
-            self.payload["cmd"]["qsh"] = self.command
+            self.payload["cmd"]["qsh"] = "system " + self.command
```

Comment:
I believe this will fail with `CLCommand('wrkactjob sbs(qinter)')`. I'd use `shlex.quote(self.command)` instead https://docs.python.org/3/library/shlex.html#shlex.quote. We can open an issue for this later if you like.

Reply:
Open an issue, please.